[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] adding connection to [http://192.168.0.101:42677/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] adding connection to [http://192.168.0.101:38227/], count 1
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] adding connection to [http://192.168.0.101:37379/], count 2
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] adding connection to [http://192.168.0.101:39031/], count 2
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf_static] adding connection to [http://192.168.0.101:42677/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] adding connection to [http://192.168.0.101:37845/], count 2
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] adding connection to [http://192.168.0.101:37517/], count 2
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] adding connection to [http://192.168.0.101:39701/], count 2
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] adding connection to [http://192.168.0.101:35045/], count 2
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] adding connection to [http://192.168.0.101:39003/], count 6
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] adding connection to [http://192.168.0.101:36287/], count 6
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] adding connection to [http://192.168.0.101:43073/], count 10
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/arm_trajectory_controller/follow_joint_trajectory/result] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/arm_trajectory_controller/follow_joint_trajectory/feedback] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/arm_trajectory_controller/follow_joint_trajectory/status] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/arm_trajectory_controller/follow_joint_trajectory/goal] adding connection to [/gazebo], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/arm_trajectory_controller/follow_joint_trajectory/cancel] adding connection to [/gazebo], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/head_trajectory_controller/follow_joint_trajectory/feedback] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/head_trajectory_controller/follow_joint_trajectory/status] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/head_trajectory_controller/follow_joint_trajectory/result] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/head_trajectory_controller/follow_joint_trajectory/cancel] adding connection to [/gazebo], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/head_trajectory_controller/follow_joint_trajectory/goal] adding connection to [/gazebo], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/follow_joint_trajectory/feedback] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/follow_joint_trajectory/status] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/follow_joint_trajectory/result] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/follow_joint_trajectory/goal] adding connection to [/gazebo], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/follow_joint_trajectory/cancel] adding connection to [/gazebo], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/omni_base_controller/follow_joint_trajectory/feedback] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/omni_base_controller/follow_joint_trajectory/result] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/omni_base_controller/follow_joint_trajectory/status] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/omni_base_controller/follow_joint_trajectory/goal] adding connection to [/gazebo], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/omni_base_controller/follow_joint_trajectory/cancel] adding connection to [/gazebo], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/impedance_control/follow_joint_trajectory/feedback] adding connection to [http://192.168.0.101:42755/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/impedance_control/follow_joint_trajectory/status] adding connection to [http://192.168.0.101:42755/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/impedance_control/follow_joint_trajectory/result] adding connection to [http://192.168.0.101:42755/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/impedance_control/follow_joint_trajectory/cancel] adding connection to [/hsrb/impedance_control], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/impedance_control/follow_joint_trajectory/goal] adding connection to [/hsrb/impedance_control], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/joint_states] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/move_base/move/feedback] adding connection to [http://192.168.0.101:40733/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/move_base/move/result] adding connection to [http://192.168.0.101:40733/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/move_base/move/status] adding connection to [http://192.168.0.101:40733/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/move_base/move/cancel] adding connection to [/move_base], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/move_base/move/goal] adding connection to [/move_base], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/apply_force/feedback] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/apply_force/result] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/grasp/feedback] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/apply_force/status] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/grasp/result] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/grasp/status] adding connection to [http://192.168.0.101:39701/], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/grasp/goal] adding connection to [/gazebo], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/apply_force/cancel] adding connection to [/gazebo], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/apply_force/goal] adding connection to [/gazebo], count 0
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/grasp/cancel] adding connection to [/gazebo], count 0
[32mINFO    [0m rospy.core:core.py:454 signal_shutdown [signal-2]
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/omni_base_controller/follow_joint_trajectory/cancel] removing connection to /gazebo
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/omni_base_controller/follow_joint_trajectory/goal] removing connection to /gazebo
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/arm_trajectory_controller/follow_joint_trajectory/goal] removing connection to /gazebo
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/grasp/cancel] removing connection to /gazebo
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/impedance_control/follow_joint_trajectory/goal] removing connection to /hsrb/impedance_control
[32mINFO    [0m rospy.internal:core.py:138 topic[/move_base/move/cancel] removing connection to /move_base
[32mINFO    [0m rospy.internal:core.py:138 topic[/rosout] removing connection to /rosout
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/head_trajectory_controller/follow_joint_trajectory/cancel] removing connection to /gazebo
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/grasp/goal] removing connection to /gazebo
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/arm_trajectory_controller/follow_joint_trajectory/cancel] removing connection to /gazebo
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/impedance_control/follow_joint_trajectory/cancel] removing connection to /hsrb/impedance_control
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/apply_force/cancel] removing connection to /gazebo
[32mINFO    [0m rospy.internal:core.py:138 topic[/move_base/move/goal] removing connection to /move_base
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/follow_joint_trajectory/goal] removing connection to /gazebo
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/head_trajectory_controller/follow_joint_trajectory/goal] removing connection to /gazebo
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/follow_joint_trajectory/cancel] removing connection to /gazebo
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/apply_force/goal] removing connection to /gazebo
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/follow_joint_trajectory/status] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/omni_base_controller/follow_joint_trajectory/result] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/apply_force/feedback] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/grasp/result] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/omni_base_controller/follow_joint_trajectory/feedback] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] removing connection to http://192.168.0.101:42677/
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] removing connection to http://192.168.0.101:38227/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:34357 (http://192.168.0.101:42677/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] removing connection to http://192.168.0.101:37379/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:33103 (http://192.168.0.101:37379/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 651, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[1m[31mERROR   [0m rospy.topics:topics.py:329 Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 839, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] removing connection to http://192.168.0.101:39031/
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] removing connection to http://192.168.0.101:37845/
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] removing connection to http://192.168.0.101:37517/
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] removing connection to http://192.168.0.101:35045/
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] removing connection to http://192.168.0.101:39003/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:56545 (http://192.168.0.101:35045/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] removing connection to http://192.168.0.101:36287/
[32mINFO    [0m rospy.internal:core.py:138 topic[/tf] removing connection to http://192.168.0.101:43073/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:37997 (http://192.168.0.101:36287/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/omni_base_controller/state] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/apply_force/status] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/grasp/feedback] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/move_base/move/feedback] removing connection to http://192.168.0.101:40733/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:46543 (http://192.168.0.101:40733/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:34357 (http://192.168.0.101:42677/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[32mINFO    [0m rospy.internal:core.py:138 topic[/tf_static] removing connection to http://192.168.0.101:42677/
[32mINFO    [0m rospy.internal:core.py:138 topic[/move_base/move/result] removing connection to http://192.168.0.101:40733/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:46543 (http://192.168.0.101:40733/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/arm_trajectory_controller/follow_joint_trajectory/feedback] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/impedance_control/follow_joint_trajectory/status] removing connection to http://192.168.0.101:42755/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/impedance_control/follow_joint_trajectory/feedback] removing connection to http://192.168.0.101:42755/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:42447 (http://192.168.0.101:42755/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/head_trajectory_controller/follow_joint_trajectory/status] removing connection to http://192.168.0.101:39701/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:57281 (http://192.168.0.101:39701/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[32mINFO    [0m rospy.internal:core.py:138 topic[/move_base/move/status] removing connection to http://192.168.0.101:40733/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/apply_force/result] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/head_trajectory_controller/follow_joint_trajectory/feedback] removing connection to http://192.168.0.101:39701/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:57281 (http://192.168.0.101:39701/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:57281 (http://192.168.0.101:39701/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 651, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/follow_joint_trajectory/result] removing connection to http://192.168.0.101:39701/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:57281 (http://192.168.0.101:39701/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[1m[31mERROR   [0m rospy.topics:topics.py:329 Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 839, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/omni_base_controller/follow_joint_trajectory/status] removing connection to http://192.168.0.101:39701/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:57281 (http://192.168.0.101:39701/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/arm_trajectory_controller/follow_joint_trajectory/status] removing connection to http://192.168.0.101:39701/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:57281 (http://192.168.0.101:39701/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 651, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/follow_joint_trajectory/feedback] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/joint_states] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/gripper_controller/grasp/status] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/head_trajectory_controller/follow_joint_trajectory/result] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/clock] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/arm_trajectory_controller/follow_joint_trajectory/result] removing connection to http://192.168.0.101:39701/
[32mINFO    [0m rospy.internal:core.py:138 topic[/hsrb/impedance_control/follow_joint_trajectory/result] removing connection to http://192.168.0.101:42755/
[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:42447 (http://192.168.0.101:42755/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[33mWARNING [0m rospy.internal:core.py:144 Unknown error initiating TCP/IP socket to 192.168.0.101:57281 (http://192.168.0.101:39701/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 559, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[32mINFO    [0m rospy.impl.masterslave:masterslave.py:342 signal-2